parm/archive/enkf_restarta_grp.yaml.j2

```
@@ -72,9 +74,9 @@ enkf_restarta_grp:
                     {% set rst_delta = "+6H" | to_timedelta %}
                 {% endif %} # DOIAU_ENKF
                 {% set rst_time = current_cycle | add_to_datetime(rst_delta) %}
-        - "{{ COMIN_OCEAN_RESTART_MEM | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ rst_time | strftime("%H") }}0000.MOM.r*.nc"
-        - "{{ COMIN_ICE_RESTART_MEM | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ rst_time | strftime("%H") }}0000.cice_model.res.nc"
         - "{{ COMIN_MED_RESTART_MEM | relpath(ROTDIR) }}/{{ cycle_YMD }}.{{ rst_time | strftime("%H") }}0000.ufs.cpld.cpl.r.nc"
+        - "{{ COMIN_OCEAN_ANALYSIS_MEM | relpath(ROTDIR) }}/{{ head }}mom6_increment.i006.nc"
+        - "{{ COMIN_ICE_ANALYSIS_MEM | relpath(ROTDIR) }}/{{ anl_time | to_YMD }}.{{ anl_time | strftime("%H") }}0000.analysis.cice_model.res.nc"
```

```suggestion
        - "{{ COMIN_ICE_ANALYSIS_MEM | relpath(ROTDIR) }}/{{ anl_time | to_fv3time }}.analysis.cice_model.res.nc"
```
`to_fv3time` returns `YYYYmmdd.HHMMSS`